src/main/scala/loamstream/model/execute/ExecutionCell.scala

```diff
@@ -0,0 +1,52 @@
+package loamstream.model.execute
+
+import loamstream.model.jobs.LJob
+import loamstream.model.jobs.JobStatus
+import loamstream.model.jobs.JobResult
+
+/**
+ * @author clint
+ * Jan 24, 2020
+ * 
+ * A snapshot of an LJob's state at a particular moment during execution, from the perspective of RxExecuter and
+ * ExecutionState.  Tracks the jobs status and the number of times it has ran. 
+ */
+final case class ExecutionCell(status: JobStatus = JobStatus.NotStarted, runCount: Int = 0) {
+  
+  def startRunning: ExecutionCell = copy(status = JobStatus.Running, runCount = runCount + 1)
```

Comment:
This feels wrong to me (naming-wise). Maybe it's "private" to the `ExecutionState` object as in no other code would ever call it. But, `startRunning` reads like a verb instead of `setAsRunning` or `markAsRunning` or similar, meaning no side-effect will take place as a result of calling this.

Reply:
I understand, I'll make it `markAsRunning`.